Checkpoint and Save Points

gamingdevotee opened the pull request:
This file makes a safe zone in which the enemy cannot enter.

gamingdevotee committed:
Add files via upload

This file makes a safe zone in which the enemy cannot enter.

gamingdevotee committed:
Add files via upload

These codes are used to set up checkpoints and save points using the player and other assets.

notleeandrew commented:
2 stars:
- it works very well
- im glad it works with a key press so if you make a mistake and cannot recover, you can just press a button and it will respawn you

wish:
- i wish you had described what I had to be attached to

gamingdevotee committed:
Add files via upload

This code is designed to allow player swap among multiple characters and cameras (two at the moment) though I am sure it can become more should you add other stuff to the array.
